.circleci/config.yml

```diff
@@ -243,6 +243,16 @@ jobs:
       - run:
           name: Set NPM authentication.
           command: echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > ~/.npmrc
+      - run:
+          # This is needed because `npm publish` invokes `npm prepare`
+          # that invokes `npm run compile` to build the Javascript code.
+          # However, compilation cannot succeed if the build tools have
+          # not been installed.  Since the build tools are dev dependencies,
+          # NODE_ENV is set to `development`.
+          name: Install modules and dependencies.
+          command: npm install
+          environment:
+            NODE_ENV: development
```

Comment:
IMO, this environment is not needed. Default behavior for npm is to install dev dependencies if you run it in an immediate project. Put differently, you don't have this environment variable set on your local machine, yet `npm install` works.

Reply:
I wasn't sure if it was needed, but thought it wouldn't hurt.  I've removed it.